main_2024.py

```diff
@@ -375,6 +418,16 @@ def main() -> int:
                     "geolocation confidence: " + str(detection_world.confidence), True
                 )
 
+        try:
+            cluster_estimations = cluster_estimation_to_main_queue.queue.get_nowait()
+        except queue.Empty:
+            cluster_estimations = None
+        if cluster_estimations is not None:
```

Comment:
Add a space above.

Reply:
pylint is telling me to not add a blank line below cluster_estimations = None